contracts/apps/accounting/FinanceApp.sol

```diff
@@ -0,0 +1,432 @@
+pragma solidity ^0.4.15;
+
+import "../App.sol";
+import "../../common/Initializable.sol";
+import "zeppelin-solidity/contracts/token/ERC20.sol";
+import "../vault/Vault.sol";
+import "../../misc/Crontab.sol";
+
+
+
+contract FinanceApp is App, Initializable, Crontab {
+
+    Vault vault;
+
+    struct AccountingPeriod {
+
+        bytes2 ct_sec;
+        bytes2 ct_min;
+        bytes2 ct_hour;
+        bytes2 ct_day;
+        bytes2 ct_month;
+        bytes2 ct_weekday;
+        bytes2 ct_year;
+        uint startBlock;
+        uint startTimestamp;
+        uint endTimestamp;
```

Comment:
By not specifying the uint size the compiler will default to using `uint256` for timestamps and block numbers that could be stored in `uint64`.